test/suites/integration/ami_test.go

```diff
@@ -78,6 +78,25 @@ var _ = Describe("LaunchTemplates", func() {
 
 		env.ExpectInstance(pod.Spec.NodeName).To(HaveField("ImageId", HaveValue(Equal(customAMI))))
 	})
+	It("should use the most recent AMI when discovering multiple", func() {
```

Comment:
I think you should move your `name` and `aws-ids` querying tests to here, personally so that we get the full E2E on querying this functionality.

Reply:
Synced up offline, this requires changing some of the artifacts that we manage in the test infrastructure. Going to change this in a future PR.